CHANGELOG.md

```diff
@@ -2,12 +2,14 @@
 # Change Log
 All notable changes to COMMIT will be documented in this file.
 
-## [1.4.5] - 2020-12-29
+## [1.4.5] - 2021-02-08
 
 ### Fixed
 - operator.pyxbld: Changed the condition to create a new operator
 - trk2dictionary.pyx: Check that the tractogram exists before trying to 
             load it and remove the try section
+- trk2dictionary.run(): fixed bug in computing the blur
+- trk2dictionary.run(): fixed bug with blur parameters
```

Comment:
Can you merge these two lies into one (they refer to the same fix)

Reply:
Yes, done